src/internal/EnhancedButton.js

```diff
@@ -296,7 +296,7 @@ class EnhancedButton extends Component {
       display: 'inline-block',
       fontFamily: this.context.muiTheme.baseTheme.fontFamily,
       WebkitTapHighlightColor: enhancedButton.tapHighlightColor, // Remove mobile color flashing (deprecated)
-      cursor: disabled ? 'default' : 'pointer',
+      cursor: disabled ? 'not-allowed' : 'pointer',
```

Comment:
`EnhancedButton` is core component and many other components rely on it.
I think it will be better to change one component once rather than change (and maintain in future) many dependent components.
Still disagree that `default` will be better.
In this case: why, do you think, original issue was created?